projects/debug_test/src/main.c

```diff
@@ -0,0 +1,69 @@
+#include <stdint.h>
```

Comment:
Not to sure what's going on with these projects? this pull request should just be the FW 102 content